workflow/snakemake/Snakefile

```diff
@@ -0,0 +1,35 @@
+# Note that if you are working on the analysis development locally, i.e. outside
+# of the REANA platform, you can proceed as follows:
+#
+#   $ cd reana-demo-worldpopulation
+#   $ mkdir snakemake-local-run
+#   $ cd snakemake-local-run
+#   $ virtualenv ~/.virtualenvs/worldpopulation-snakemake
+#   $ source ~/.virtualenvs/worldpopulation-snakemake/bin/activate
+#   $ pip install snakemake
+#   $ cp -a ../code ../data .
+#   $ snakemake -s ../workflow/snakemake/Snakefile \
+#               --configfile ../workflow/snakemake/inputs.yaml -p --cores 1
+#   $ open results/plot.png
+
+rule all:
+    input:
+        config["notebook"],
+        config["input_file"]
+
+rule worldpopulation:
+    input:
+        notebook=config["notebook"],
+        input_file=config["input_file"]
+    params:
+        region=config["region"],
+        year_max=config["year_max"],
+        year_min=config["year_min"]
+    output:
+        config["output_file"]
+    container:
+        "reanahub/reana-env-jupyter:1.0.0"
+    shell:
+        "mkdir -p results && papermill ${input.notebook} /dev/null -p "
+        "input_file ${input.input_file} -p output_file ${output.output_file} "
+        "-p region ${params.region} -p year_min ${params.year_min} -p year_max ${params.year_max}"
```

Comment:
After some testing I noticed that Snakemake does not need `$` for params, it should be `{input.input_file}` instead of `${input.input_file}`